tests/gnrc_udp/udp.c

```diff
@@ -256,3 +259,4 @@ int udp_cmd(int argc, char **argv)
     }
     return 0;
 }
+
```

Comment:
You are introducing an additional newline here.

Reply:
Thank you for your prompt feedback, appreciate it very much. I have reopened the pull-request by inculcating your changes.